generators/app/templates/server/config/application.rb

```diff
@@ -7,14 +7,15 @@
 require "active_record/railtie"
 require "action_controller/railtie"
 require "action_mailer/railtie"
-# require "action_view/railtie"
-# require "action_cable/engine"
+require "action_view/railtie"
+require "action_cable/engine"
 # require "sprockets/railtie"
-# require "rails/test_unit/railtie"
+require "rails/test_unit/railtie"
```

Comment:
I don't really know. I have to admit I just tried to keep it lightweight commenting stuff that didn't seem to matter when using Rails only as an API backend, but I didn't really benchmarked anything or tested if it really made an impact.